packages/neuron-ui/src/components/TransactionFeePanel/index.tsx

```diff
@@ -79,7 +84,7 @@ const TransactionFee: React.FunctionComponent<TransactionFee> = ({
             <Label>{t('send.price')}</Label>
           </Stack.Item>
           <Stack.Item grow>
-            <TextField value={price} onChange={onPriceChange} aria-label="price" />
+            <TextField value={price} onChange={onPriceChange} aria-label="price" suffix="shannons/kB" />
```

Comment:
```suggestion
            <TextField value={price} onChange={onPriceChange} aria-label="price" suffix="shannons/KB" />
```

Reply:
I'd like to say, `kB` is more precise.
![image](https://user-images.githubusercontent.com/7271329/67383522-df93a800-f5c1-11e9-8923-5b8d91fc89c9.png)

Reply:
ok